src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
A note on this: In the ES6 classes PR #1993 I've only converted Component so far, and all tests pass after adding back `init` and `Class.extend`. So it is definitely possible to add support for the old inheritance API, we just need to decide if that should be included in the core lib (with deprecation warnings) or only in a migration plugin. I think I'd be tempted to add it to core, and then remove it in a later minor release.